docs/deployment/plugins/k8s/index.rst

```diff
@@ -277,6 +277,85 @@ Spin up a cluster
 
     helm repo add flyteorg https://flyteorg.github.io/flyte
 
+  .. group-tab:: Flyte sandbox
+
+    If you have installed Flyte using the `flyte-core Helm chart
+    <https://github.com/flyteorg/flyte/tree/master/charts/flyte-core>`__, please ensure:
+
+    * You have the correct kubeconfig and have selected the correct Kubernetes context.
+    * You have configured the correct flytectl settings in ``~/.flyte/config.yaml``.
+
+    .. tabs::
+
+      .. group-tab:: Helm chart
+
+        .. tabs::
+
+          .. group-tab:: Spark
+
+            create the following four files and apply them using ``kubectl apply -f <filename>``:
+
+            1. ``serviceaccount.yaml``
+
+            .. code-block:: yaml
+
+              apiVersion: v1
+              kind: ServiceAccount
+              metadata:
+                name: default
+                namespace: "{{ namespace }}"
+                annotations:
+                  eks.amazonaws.com/role-arn: "{{ defaultIamRole }}"
+
+            2. ``spark_role.yaml``
+
+            .. code-block:: yaml
+
+              apiVersion: rbac.authorization.k8s.io/v1
+              kind: Role
+              metadata:
+                name: spark-role
+                namespace: "{{ namespace }}"
+              rules:
```

Comment:
One of the tests is complaining about a list being out of range. Not sure if it's a matter of indentation.

Also, this is the Role I've been using with flyte-core on EKS:

https://github.com/flyteorg/flyte/pull/4670/files#diff-5e363febf4c9f8a60b6f884277d82eb90f1d76e1f2a5b14037133c1c9ff36adeR574-R601